runtime/doc/treesitter.txt

```diff
@@ -164,11 +164,28 @@ Tree-sitter queries are supported, with some limitations. Currently, the only
 supported match predicate is `eq?` (both comparing a capture against a string
 and two captures against each other).
 
+A |query| consists of one or more patterns. A |pattern| is defined over node
```

Comment:
We probably want to do prefix common words as tags when used for tree-sitter specifically

Reply:
Good point.  In that case I think it's better to switch from tags to code.  Rewriting |query| --> |lua-treesitter-query| or |tsquery| implies the description is describing code in treesitter. But really it's just meant as a brief overview of tree-sitter terms that are used in code below but not defined in vim docs (or explicitly defined at treesitter docs.) PTAL.